### PR TITLE
SignUp form added

### PR DESCRIPTION
**Pull request by HassanMostafaa**

### CODE REVIEW

File: css/login.css [hunk 1]
```diff
@@ -118,13 +118,15 @@ input:focus {
 .forget-pass {
   font-weight: 500;
   margin-top: 15px;
-
```
Comment (Mohamed-98) [inline css/login.css:111]: ### WTF

File: css/style.css [hunk 2]
```diff
@@ -1,5 +1,5 @@
 body {
-  background-image: url(../img/bgO.png);
```
Comment (Mohamed-98) [inline css/style.css:2]: WHY

File: css/style.css [hunk 3]
```diff
@@ -9,22 +9,36 @@ nav {
...
-  font-size: 18px;
+  font-size: 20px;
   overflow: hidden;
+  padding: 12px 15px;
+  line-height: 1.4rem;
+  transition: all 0.5s;
 }
 nav li a:hover {
-  background: rgba(255, 255, 255, 0);
-
```
Comment (Mohamed-98) [inline css/style.css:22]: What is this

File: css/style.css [hunk 4]
```diff
@@ -9,22 +9,36 @@ nav {
...
+  color: #f4f4f4 !important;
+  border-radius: 30px;
+  outline: none;
+  transition: all 0.5s;
   color: #6344a4;
 }
 .active {
+  background: #6344a4;
   position: relative;
-  color: #6344a4;
```
Comment (Mohamed-98) [inline css/style.css:27]: why

**Review by Mohamed-98 — COMMENTED** [review]
check the comments

File: css/style.css (hunk 4 above)
Comment (HassanMostafaa) [reply css/style.css:27]: because

**Review by Mohamed-98 — APPROVED** [review]
i dont feel it but its oky

**Pull request merged**